modules/oadp-plugins.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+[id="oadp-plugins_{context}"]
+= OADP plug-ins
+
+Velero uses plug-ins that are integrated with storage providers. These plug-ins support backup and snapshot operations.
+
+The OpenShift API for Data Protection (OADP) provides default OADP plug-ins that contain Velero plug-ins for specific storage locations. You can create link:https://velero.io/docs/v1.7/custom-plugins/[custom plug-ins] based on the default plug-ins.
+
+[NOTE]
+====
+An OADP plug-in, for example, `aws`, can contain multiple plug-ins.
+====
+
+[cols="3", options="header"]
+.OADP default plug-ins
```

Comment:
This table could still be clearer. After reading around the doc, here's what I understand. The aws plugin, for example, contains two plugins, which are described in col. 2. These two plugins back up different types of data and store that data in different types of storage.

The table tries to answer these questions: What are the default plugins (col. 1)? Where do they put my stuff (cols. 2-3)?

Given that, do we really need to know more about these subsidiary plugins?

Perhaps we can reimagine column 2. Don't describe the HOW. Describe the WHAT.

| plug-in | backup file types | backup storage locations | ( WHAT plugin?, WHAT stuff? , WHERE?)

In col 2, change the descriptions to a list of what's being backed up.
For example:
`Volume snapshot plug-in for backing up and restoring volumes with snapshots.`
Changes to
`snapshots`

Reply:
Alternatively, if it's better to provide a list of the Valero plugins: Could we add specific Valero plugin names to the column?

Reply:
> Alternatively, if it's better to provide a list of the Valero plugins: Could we add specific Valero plugin names to the column?

The Velero plugins have the same name as OADP plugins. aws = aws. azure = azure. Each Velero plugin contains two plugins.

Reply:
> | plug-in | backup file types | backup storage locations | ( WHAT plugin?, WHAT stuff? , WHERE?)

Definitely not "backup storage locations". That is a specific technical term, only used for backups of k8s resources. It does not include backups of PVs, for example.

There are two types of 'backups': backups (yes, that really is what it's called) and snapshots. (Unless you're using Restic, but that does not have a plugin). So we have backup storage locations (BSLs) and volume storage locations (VSL). I supposed I could change the column header to "storage location". I will think about this.

Reply:
@ctauchen 
I rewrote the table. What do you think?

Reply:
@apinnick That did it! Function and Storage Location were the right choices.

The changes to the Function descriptions are also much clearer. To my ear, they read a bit better when the verbs are changed to gerunds. As I scan the table, I read to myself: Here's the list of Functions, and the first Function is ~~"Backs up and restores Kubernetes objects by using object store."~~ "Backing up and restoring Kubernetes objects ..."

Reply:
@ctauchen 
Whew. Thanks! I know the original text was very repetitious. It was a lot of copy/paste from the Velero upstream docs with too little time for me to rewrite, mainly because the first GA is next week. Glad that it's clearer. 

Many thanks!